src/sagas/SleepingPillApi.js

```diff
@@ -11,3 +13,17 @@ export const sessionBySlug: (string) => any =
   (slug) => fetch(`${config.urls.sleepingPill}/public/allSessions/${slug}`)
     .then((res) => res.json())
 
+const options = {
+	"Content-Type": "application/json",
+    headers: {
+      'Content-Type': "application/json",
+      "Voter-ID": DeviceInfo? "javazone2017" : "javazone2017"
+    }
+}
+export const feedback: (feedback) => any =
```

Comment:
Move this to a file named `DevNullApi.js` since it doens't interact with SleepingPill